plugins/pulp_docker/plugins/importers/upload.py

```diff
@@ -409,7 +409,16 @@ def process_main(self, item=None):
             if not checksum_type:
                 # Never assume. But oh well
                 checksum_type = "sha256"
-            blob_src_path = os.path.join(self.get_working_dir(), checksum + '.tar')
+
+            with open(os.path.join(self.get_working_dir(), 'version')) as version_file:
+                version = version_file.readline().split(':')[1].strip()
```

Comment:
Recommend encapsulating understanding of the version file format in the `Version` class and add something like:
```
@classmethod
def read(cls, path):
    with open(path) as fp:
        version = fp.readline().split(':')[1].strip()
        return cls(version)
```